pnpm-lock.yaml

```diff
@@ -22691,6 +22691,34 @@ packages:
       - debug
       - supports-color
 
+  /@fluidframework/server-lambdas@5.0.0(debug@4.3.6):
```

Comment:
Hmm none of our changes should have resulted in a new dependency on server-lambdas... I think the lockfile might need a revert and clean update?

Reply:
I realized it's adding a "new" dependency specifically for "+ `debug` 4.3.6" (the existing one uses "+ `debug` 4.3.5), because in some other bits of the lockfile, the `debug` dependency got updated 4.3.5 -> 4.3.6 which shouldn't be necessary. So I do think we want to revert the lockfile and update it again with `pnpm i`, expecting that the dependency on `debug` does not change versions.

Reply:
Seems like the lock-pnpm.yaml won't update along with the change in package.json.
Run `pnpm i` will throw error:
~~~
Scope: all 159 workspace projects
 ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up to date with packages/common/core-utils/package.json

Note that in CI environments this setting is true by default. If you still need to run install in such cases, use "pnpm install --no-frozen-lockfile"

    Failure reason:
    specifiers in the lockfile ({"@arethetypeswrong/cli":"^0.15.2","@biomejs/biome":"~1.8.3","@fluid-internal/mocha-test-setup":"workspace:~","@fluid-tools/benchmark":"^0.50.0","@fluid-tools/build-cli":"^0.42.0","@fluidframework/build-common":"^2.0.3","@fluidframework/build-tools":"^0.42.0","@fluidframework/core-utils-previous":"npm:@fluidframework/core-utils@2.1.0","@fluidframework/eslint-config-fluid":"^5.3.0","@microsoft/api-extractor":"^7.45.1","@types/mocha":"^9.1.1","@types/node":"^18.19.0","@types/sinon":"^17.0.3","c8":"^8.0.1","concurrently":"^8.2.1","copyfiles":"^2.4.1","cross-env":"^7.0.3","eslint":"~8.55.0","eslint-config-prettier":"~9.0.0","mocha":"^10.2.0","mocha-json-output-reporter":"^2.0.1","mocha-multi-reporters":"^1.5.1","moment":"^2.21.0","prettier":"~3.0.3","rimraf":"^4.4.0","sinon":"^17.0.1","typescript":"~5.4.5"}) don't match specs in package.json ({"@arethetypeswrong/cli":"^0.15.2","@biomejs/biome":"~1.8.3","@fluid-internal/mocha-test-setup":"workspace:~","@fluid-tools/benchmark":"^0.47.0","@fluid-tools/build-cli":"^0.42.0","@fluidframework/build-common":"^2.0.3","@fluidframework/build-tools":"^0.42.0","@fluidframework/core-utils-previous":"npm:@fluidframework/core-utils@2.1.0","@fluidframework/eslint-config-fluid":"^5.3.0","@microsoft/api-extractor":"^7.45.1","@types/mocha":"^9.1.1","@types/node":"^18.19.0","@types/sinon":"^17.0.3","c8":"^8.0.1","concurrently":"^8.2.1","copyfiles":"^2.4.1","cross-env":"^7.0.3","eslint":"~8.55.0","eslint-config-prettier":"~9.0.0","mocha":"^10.2.0","mocha-json-output-reporter":"^2.0.1","mocha-multi-reporters":"^1.5.1","moment":"^2.21.0","prettier":"~3.0.3","rimraf":"^4.4.0","sinon":"^17.0.1","typescript":"~5.4.5"})
~~~

and after run `pnpm install --no-frozen-lockfile` was it recommand will add dependency automatedly. Updating pnpm-lock.yaml manually...

Reply:
You should not need to update the lockfile manually. What's making you need to do that?